slack_bolt/listener/async_internals.py

```diff
@@ -0,0 +1,59 @@
+from logging import Logger
+from typing import Dict, Any, Optional
+
+from slack_bolt.request.async_request import AsyncBoltRequest
+from slack_bolt.request.payload_utils import (
+    to_options,
+    to_shortcut,
+    to_action,
+    to_view,
+    to_command,
+    to_event,
+    to_message,
+    to_step,
+)
+from slack_bolt.response import BoltResponse
+
+
+def _build_all_available_args(
+    logger: Logger,
+    request: AsyncBoltRequest,
+    response: Optional[BoltResponse],
+    error: Optional[Exception] = None,
+) -> Dict[str, Any]:
+    all_available_args = {
+        "logger": logger,
+        "error": error,
+        "client": request.context.client,
+        "req": request,
+        "request": request,
+        "resp": response,
+        "response": response,
+        "context": request.context,
+        "body": request.body,
+        # payload
+        "body": request.body,
```

Comment:
Looks like `body` was declared twice (I'm guessing the `#payload` comment is meant to include body):
```suggestion
        # payload
        "body": request.body,
```

Reply:
@mwbrooks Thanks for catching these!